frontend/src/metabase/query_builder/components/expressions/ExpressionWidget.tsx

```diff
@@ -1,16 +1,27 @@
 import React, { useRef, useState } from "react";
-import cx from "classnames";
 import { t } from "ttag";
 import { isNotNull } from "metabase/core/utils/types";
-import { ExpressionWidgetWrapper } from "metabase/query_builder/components/expressions/ExpressionWidget.styled";
-import ExternalLink from "metabase/core/components/ExternalLink";
-import MetabaseSettings from "metabase/lib/settings";
-import { Expression } from "metabase-types/types/Query";
+import {
+  Divider,
+  ExpressionWidgetButtonsWrapper,
+  ExpressionWidgetExpressionFieldWrapper,
+  ExpressionWidgetFieldTitle,
+  ExpressionWidgetFieldWrapper,
+  ExpressionWidgetFooter,
+  ExpressionWidgetIconWrapper,
+  ExpressionWidgetInput,
+  ExpressionWidgetRemoveLink,
+  ExpressionWidgetStyledIcon,
+  ExpressionWidgetWrapper,
+} from "metabase/query_builder/components/expressions/ExpressionWidget.styled";
```

Comment:
Should this be a local import?

Reply:
@deniskaber why do you use names like this? We usually do not make styled components name s so unique. E.g. https://github.com/metabase/metabase/blob/c9fbddc2e8c9cf86d8c100c1237bd12deb9a72f5/frontend/src/metabase/models/components/ModelDetailPage/ModelActionDetails/ModelActionListItem.styled.tsx#L79